<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>requester</class>
 <widget class="QDialog" name="requester">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1381</width>
    <height>784</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>Carlito</family>
    <pointsize>26</pointsize>
    <bold>true</bold>
   </font>
  </property>
  <property name="windowTitle">
   <string>Solicitante - SST</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout" stretch="2,5">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QWidget" name="leftMenuContainer" native="true">
     <property name="styleSheet">
      <string notr="true">background: #3383ff;</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_3" stretch="1,5,1">
      <property name="spacing">
       <number>50</number>
      </property>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_2" stretch="1,1,1">
        <property name="spacing">
         <number>2</number>
        </property>
        <item>
         <widget class="QLabel" name="label">
          <property name="font">
           <font>
            <family>Carlito</family>
            <pointsize>72</pointsize>
            <bold>true</bold>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">color: #f9f9f9;</string>
          </property>
          <property name="text">
           <string>SST</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignmentFlag::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_3">
          <property name="font">
           <font>
            <family>Carlito</family>
            <pointsize>16</pointsize>
            <bold>true</bold>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">color: #f9f9f9;</string>
          </property>
          <property name="text">
           <string>Solicitante</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignmentFlag::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="requesterUserName">
          <property name="font">
           <font>
            <family>Carlito</family>
            <pointsize>24</pointsize>
            <bold>true</bold>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">color: #f9f9f9;</string>
          </property>
          <property name="text">
           <string>Usuário</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignmentFlag::AlignCenter</set>
          </property>
          <property name="wordWrap">
           <bool>true</bool>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout">
        <property name="spacing">
         <number>0</number>
        </property>
        <item>
         <widget class="HoverWidget" name="requesterCallsBtn">
          <property name="styleSheet">
           <string notr="true">background: grey;
border-radius: 5px;</string>
          </property>
          <property name="text">
           <string>PushButton</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="HoverWidget" name="logoutRequesterBtn">
          <property name="styleSheet">
           <string notr="true">background: grey;
border-radius: 5px;</string>
          </property>
          <property name="text">
           <string>PushButton</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <spacer name="verticalSpacer">
        <property name="orientation">
         <enum>Qt::Orientation::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QStackedWidget" name="main">
     <property name="styleSheet">
      <string notr="true">background: #f9f9f9;</string>
     </property>
     <widget class="QWidget" name="mainWindow">
      <layout class="QGridLayout" name="gridLayout">
       <item row="0" column="0">
        <widget class="QWidget" name="widget_2" native="true">
         <layout class="QVBoxLayout" name="verticalLayout_4" stretch="2,2,4">
          <property name="spacing">
           <number>2</number>
          </property>
          <property name="leftMargin">
           <number>30</number>
          </property>
          <property name="topMargin">
           <number>30</number>
          </property>
          <property name="rightMargin">
           <number>30</number>
          </property>
          <property name="bottomMargin">
           <number>30</number>
          </property>
          <item>
           <widget class="QLabel" name="label_5">
            <property name="font">
             <font>
              <family>Carlito</family>
              <pointsize>72</pointsize>
              <bold>true</bold>
             </font>
            </property>
            <property name="styleSheet">
             <string notr="true">color: #3383ff;</string>
            </property>
            <property name="text">
             <string>Sistema de Suporte Técnico</string>
            </property>
            <property name="wordWrap">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="label_4">
            <property name="font">
             <font>
              <family>Carlito</family>
              <pointsize>24</pointsize>
              <bold>true</bold>
             </font>
            </property>
            <property name="styleSheet">
             <string notr="true">color: #3383ff;</string>
            </property>
            <property name="text">
             <string>Precisa de ajuda? Abra um chamado no botão abaixo e aguarde nossa equipe!</string>
            </property>
            <property name="wordWrap">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="HoverWidget" name="openCallBtn">
            <property name="styleSheet">
             <string notr="true">background: grey;
border-radius: 5px;</string>
            </property>
            <property name="text">
             <string>PushButton</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="createCall">
      <layout class="QGridLayout" name="gridLayout_2">
       <item row="0" column="0">
        <widget class="QWidget" name="widget_3" native="true">
         <layout class="QVBoxLayout" name="verticalLayout_7" stretch="1,1,4,1,1">
          <property name="spacing">
           <number>30</number>
          </property>
          <property name="leftMargin">
           <number>10</number>
          </property>
          <property name="topMargin">
           <number>10</number>
          </property>
          <property name="rightMargin">
           <number>10</number>
          </property>
          <property name="bottomMargin">
           <number>10</number>
          </property>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_3" stretch="2,9,2">
            <item>
             <widget class="QLabel" name="label_6">
              <property name="font">
               <font>
                <family>Carlito</family>
                <pointsize>48</pointsize>
                <bold>true</bold>
               </font>
              </property>
              <property name="styleSheet">
               <string notr="true">color: #3383ff;</string>
              </property>
              <property name="text">
               <string>Novo Chamado</string>
              </property>
             </widget>
            </item>
            <item>
             <spacer name="horizontalSpacer_2">
              <property name="orientation">
               <enum>Qt::Orientation::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
            <item>
             <widget class="HoverWidget" name="goBackNewCallBtn">
              <property name="styleSheet">
               <string notr="true">background: grey;
border-radius: 5px;</string>
              </property>
              <property name="text">
               <string>PushButton</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_4" stretch="1,10">
            <item>
             <widget class="QLabel" name="label_7">
              <property name="font">
               <font>
                <family>Carlito</family>
                <pointsize>24</pointsize>
                <bold>true</bold>
               </font>
              </property>
              <property name="styleSheet">
               <string notr="true">color: #3383ff;</string>
              </property>
              <property name="text">
               <string>Título</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="titleLineEdit">
              <property name="font">
               <font>
                <family>Carlito</family>
                <pointsize>18</pointsize>
               </font>
              </property>
              <property name="styleSheet">
               <string notr="true">border: 1px solid #3383ff;
border-radius: 5px;
color: #3383ff;
padding: 10px;</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QVBoxLayout" name="verticalLayout_6">
            <property name="spacing">
             <number>2</number>
            </property>
            <item>
             <widget class="QLabel" name="label_10">
              <property name="font">
               <font>
                <family>Carlito</family>
                <pointsize>24</pointsize>
                <bold>true</bold>
               </font>
              </property>
              <property name="styleSheet">
               <string notr="true">color: #3383ff;</string>
              </property>
              <property name="text">
               <string>Descrição</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QTextEdit" name="descTextEdit">
              <property name="font">
               <font>
                <family>Carlito</family>
                <pointsize>18</pointsize>
               </font>
              </property>
              <property name="styleSheet">
               <string notr="true">border: 1px solid #3383ff;
border-radius: 5px;
color: #3383ff;
padding: 10px</string>
              </property>
              <property name="html">
               <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;meta charset=&quot;utf-8&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
hr { height: 1px; border-width: 0; }
li.unchecked::marker { content: &quot;\2610&quot;; }
li.checked::marker { content: &quot;\2612&quot;; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Carlito'; font-size:18pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px; font-family:'Sans Serif'; font-size:9pt;&quot;&gt;&lt;br /&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_8">
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_5" stretch="1,10">
              <item>
               <widget class="QLabel" name="label_8">
                <property name="font">
                 <font>
                  <family>Carlito</family>
                  <pointsize>24</pointsize>
                  <bold>true</bold>
                 </font>
                </property>
                <property name="styleSheet">
                 <string notr="true">color: #3383ff;</string>
                </property>
                <property name="text">
                 <string>Tipo</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QComboBox" name="typeComboBox">
                <property name="font">
                 <font>
                  <family>Carlito</family>
                  <pointsize>18</pointsize>
                 </font>
                </property>
                <property name="styleSheet">
                 <string notr="true">border: 1px solid #3383ff;
border-radius: 5px;
color: #3383ff;
padding: 10px;</string>
                </property>
               </widget>
              </item>
             </layout>
            </item>
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_6" stretch="1,10">
              <item>
               <widget class="QLabel" name="label_9">
                <property name="font">
                 <font>
                  <family>Carlito</family>
                  <pointsize>24</pointsize>
                  <bold>true</bold>
                 </font>
                </property>
                <property name="styleSheet">
                 <string notr="true">color: #3383ff;</string>
                </property>
                <property name="text">
                 <string>Prioridade</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QComboBox" name="priorityComboBox">
                <property name="font">
                 <font>
                  <family>Carlito</family>
                  <pointsize>18</pointsize>
                 </font>
                </property>
                <property name="styleSheet">
                 <string notr="true">border: 1px solid #3383ff;
border-radius: 5px;
color: #3383ff;
padding: 10px;</string>
                </property>
               </widget>
              </item>
             </layout>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_7">
            <item>
             <spacer name="horizontalSpacer_3">
              <property name="orientation">
               <enum>Qt::Orientation::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
            <item>
             <widget class="HoverWidget" name="sendCallBtn">
              <property name="styleSheet">
               <string notr="true">background: grey;
border-radius: 5px;</string>
              </property>
              <property name="text">
               <string>PushButton</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="requesterCallsPage">
      <layout class="QGridLayout" name="gridLayout_3">
       <item row="0" column="0">
        <widget class="QWidget" name="widget" native="true">
         <layout class="QVBoxLayout" name="verticalLayout_5">
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_2" stretch="2,9,2">
            <item>
             <widget class="QLabel" name="label_2">
              <property name="font">
               <font>
                <family>Carlito</family>
                <pointsize>48</pointsize>
                <bold>true</bold>
               </font>
              </property>
              <property name="styleSheet">
               <string notr="true">color: #3383ff;</string>
              </property>
              <property name="text">
               <string>Meus Chamados</string>
              </property>
             </widget>
            </item>
            <item>
             <spacer name="horizontalSpacer">
              <property name="orientation">
               <enum>Qt::Orientation::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
            <item>
             <widget class="HoverWidget" name="goBackRequesterCallsBtn">
              <property name="styleSheet">
               <string notr="true">background: grey;
border-radius: 5px;</string>
              </property>
              <property name="text">
               <string>PushButton</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <widget class="QTableWidget" name="requesterCallsTableWidget">
            <property name="styleSheet">
             <string notr="true">border: 1px solid #3383ff;
border-radius: 5px;</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="genericCallPage">
      <layout class="QGridLayout" name="gridLayout_4">
       <item row="0" column="1">
        <widget class="QWidget" name="widget_4" native="true">
         <layout class="QVBoxLayout" name="verticalLayout_10" stretch="0,0,0,3,4">
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_9" stretch="2,9,2">
            <item>
             <widget class="QLabel" name="titleLabel">
              <property name="font">
               <font>
                <family>Carlito</family>
                <pointsize>48</pointsize>
                <bold>true</bold>
               </font>
              </property>
              <property name="styleSheet">
               <string notr="true">color: #3383ff;</string>
              </property>
              <property name="text">
               <string>Titulo Chamado</string>
              </property>
             </widget>
            </item>
            <item>
             <spacer name="horizontalSpacer_4">
              <property name="orientation">
               <enum>Qt::Orientation::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
            <item>
             <widget class="HoverWidget" name="goBackGenericCallBtn">
              <property name="styleSheet">
               <string notr="true">background: grey;
border-radius: 5px;</string>
              </property>
              <property name="text">
               <string>PushButton</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <widget class="QLabel" name="statusClosedDateLabel">
            <property name="font">
             <font>
              <family>Carlito</family>
              <pointsize>20</pointsize>
              <bold>false</bold>
             </font>
            </property>
            <property name="styleSheet">
             <string notr="true">color: #3383ff;</string>
            </property>
            <property name="text">
             <string>Status/Data_Fechamento</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="responLabel">
            <property name="font">
             <font>
              <family>Carlito</family>
              <pointsize>28</pointsize>
              <bold>true</bold>
             </font>
            </property>
            <property name="styleSheet">
             <string notr="true">color: #3383ff;</string>
            </property>
            <property name="text">
             <string>Responsável</string>
            </property>
           </widget>
          </item>
          <item>
           <layout class="QVBoxLayout" name="verticalLayout_9">
            <item>
             <widget class="QLabel" name="label_13">
              <property name="font">
               <font>
                <family>Carlito</family>
                <pointsize>28</pointsize>
                <bold>true</bold>
               </font>
              </property>
              <property name="styleSheet">
               <string notr="true">color: #3383ff;</string>
              </property>
              <property name="text">
               <string>Descrição</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QTextBrowser" name="descTextBrowser">
              <property name="font">
               <font>
                <family>Carlito</family>
                <pointsize>14</pointsize>
               </font>
              </property>
              <property name="styleSheet">
               <string notr="true">color: #3383ff;
border-radius: 5px;
border: 1px solid #3383ff;
padding: 10px;</string>
              </property>
              <property name="html">
               <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;meta charset=&quot;utf-8&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
hr { height: 1px; border-width: 0; }
li.unchecked::marker { content: &quot;\2610&quot;; }
li.checked::marker { content: &quot;\2612&quot;; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Carlito'; font-size:14pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-size:9pt;&quot;&gt;Descrição do Problema Aqui!&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QVBoxLayout" name="verticalLayout_8">
            <item>
             <widget class="QLabel" name="label_14">
              <property name="font">
               <font>
                <family>Carlito</family>
                <pointsize>28</pointsize>
                <bold>true</bold>
               </font>
              </property>
              <property name="styleSheet">
               <string notr="true">color: #3383ff;</string>
              </property>
              <property name="text">
               <string>Chat</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QListWidget" name="chatListWidget">
              <property name="font">
               <font>
                <family>Carlito</family>
                <pointsize>14</pointsize>
               </font>
              </property>
              <property name="styleSheet">
               <string notr="true">color: #3383ff;
padding: 10px;
border-radius: 5px;
border: 1px solid #3383ff;</string>
              </property>
             </widget>
            </item>
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_10">
              <item>
               <widget class="QLineEdit" name="messageLineEdit">
                <property name="font">
                 <font>
                  <family>Carlito</family>
                  <pointsize>14</pointsize>
                 </font>
                </property>
                <property name="styleSheet">
                 <string notr="true">color: #3383ff;
padding: 10px;
border-radius: 5px;
border: 1px solid #3383ff;</string>
                </property>
                <property name="placeholderText">
                 <string>Digite sua mensagem aqui...</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="HoverWidget" name="sendMessageBtn">
                <property name="styleSheet">
                 <string notr="true">background: grey;
border-radius: 5px;</string>
                </property>
                <property name="text">
                 <string>PushButton</string>
                </property>
               </widget>
              </item>
             </layout>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>HoverWidget</class>
   <extends>QPushButton</extends>
   <header>hoverwidget.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
